docs/iol/instant-v2/instant_v2.qmd

```diff
@@ -0,0 +1,58 @@
+---
+title: "Instant HIE v2"
+format: html
+---
+
+
+##### Background
+Instant OpenHIE (currently Instant OpenHIE v2) aims to allow Health Information Exchange components to be packaged up, deployed, operated and scaled via a simple CLI.
+
+
+##### MamaToto HIE
+
```

Comment:
it would be nice to describe a short summary of what each component does and include fhirweb too